dev-tools/jenkins_ci.sh

```diff
@@ -19,12 +19,18 @@ jenkins_setup
 cleanup() {
   echo "Running cleanup..."
   rm -rf $TEMP_PYTHON_ENV
-  make stop-environment || true
-  make fix-permissions || true
-  echo "Killing all running containers..."
-  docker ps -q | xargs -r docker kill || true
-  echo "Cleaning stopped docker containers and dangling images/networks/volumes..."
-  docker system prune -f || true
+
+  if docker info > /dev/null ; then
```

Comment:
How is this change related?

Reply:
It’s a separate problem. (So do a rebase and merge to keep the commits separate.) This relates to the infra issue linked above (Mac provisioning).

Reply:
Got it, didn't see that one.